LibreNMS/Modules/Os.php

```diff
@@ -65,23 +66,17 @@ public function poll(\LibreNMS\OS $os): void
         if ($os instanceof OSPolling) {
             $os->pollOS();
         } else {
-            // legacy poller files
-            global $graphs, $device;
-
-            if (empty($device)) {
-                $device = $os->getDeviceArray();
-            }
-
+            $device = $os->getDeviceArray();
             $location = null;
 
             if (is_file(base_path('/includes/polling/os/' . $device['os'] . '.inc.php'))) {
```

Comment:
Should remove the support completely instead?

Reply:
I would like to, but I know it will break a lot of people that have local os definitions.  So I decided to send a deprecation event log for now.